File: Semester 2/Research Project/2020_AON01/Stage-Frontend/src/app/classes/reviewer.ts
import {User} from './user';
import {Stagevoorstel} from '@app/classes/stagevoorstel';

export class Reviewer extends User {
  toegewezenVoorstellen: Stagevoorstel[];
  voorkeurVoorstellen: Stagevoorstel[];
  // isCoordinator: boolean;
}

File: Semester 2/Research Project/2020_AON01/Stage_API/Stage_API.Domain/Classes/Reviewer.cs
using Stage_API.Domain.Relations;
using System.Collections.Generic;

namespace Stage_API.Domain.Classes
{
    public class Reviewer : User
    {
        public ICollection<ReviewerStagevoorstelToegewezen> ToegewezenVoorstellen { get; set; }
        public ICollection<ReviewerStagevoorstelFavoriet> VoorkeurVoorstellen { get; set; }
        public ICollection<Review> Reviews { get; set; }
        public bool IsCoordinator { get; set; }
    }
}